src/Runner/Runner.php

```diff
@@ -188,30 +192,30 @@ private function fixParallel(): array
         };
 
         // [REACT] Handle worker's handshake (init connection)
-        $server->on('connection', static function (ConnectionInterface $connection) use ($processPool, $fileChunk): void {
+        $server->on('connection', static function (ConnectionInterface $connection) use ($processPool, $filesChunkGenerator): void {
             $jsonInvalidUtf8Ignore = \defined('JSON_INVALID_UTF8_IGNORE') ? JSON_INVALID_UTF8_IGNORE : 0;
             $decoder = new Decoder($connection, true, 512, $jsonInvalidUtf8Ignore);
             $encoder = new Encoder($connection, $jsonInvalidUtf8Ignore);
 
             // [REACT] Bind connection when worker's process requests "hello" action (enables 2-way communication)
-            $decoder->on('data', static function (array $data) use ($processPool, $fileChunk, $decoder, $encoder): void {
+            $decoder->on('data', static function (array $data) use ($processPool, $filesChunkGenerator, $decoder, $encoder): void {
                 if (ParallelAction::RUNNER_HELLO !== $data['action']) {
                     return;
                 }
 
                 $identifier = ProcessIdentifier::fromRaw($data['identifier']);
                 $process = $processPool->getProcess($identifier);
                 $process->bindConnection($decoder, $encoder);
-                $job = $fileChunk();
+                $filesChunkForJob = $filesChunkGenerator();
```

Comment:
Following previous comment, renaming callable-variable would allow using `$fileChunk` for actual file chunk here.

```suggestion
                $fileChunk = $getFileChunk();
```

Reply:
love it, applied